presto-main/src/main/java/com/facebook/presto/server/PrestoServer.java

```diff
@@ -138,7 +140,8 @@ public void run()
                 new TempStorageModule(),
                 new QueryPrerequisitesManagerModule(),
                 new NodeTtlFetcherManagerModule(),
-                new ClusterTtlProviderManagerModule());
+                new ClusterTtlProviderManagerModule(),
+                new ClientRequestFilterModule());
```

Comment:
Just add this inline to `HttpServerModule`, it's a one liner and strongly aligned with HTTP.